src/components/v2/values/chartValuesDiff/ChartValuesView.type.ts

```diff
@@ -285,3 +287,29 @@ export interface ChartValuesViewAction {
     type: ChartValuesViewActionTypes
     payload: any
 }
+
+export interface InputType {
+    valueName: string
+    handleValueNameChange: (newAppName: string) => void
+    handleValueNameOnBlur: () => void
+    invalidValueName: boolean
+    invalidValueNameMessage: string
+    valueNameDisabled: boolean
+}
+
+export interface AppNameInputType {
+    appName: string
+    handleAppNameChange: (newAppName: string) => void
+    handleAppNameOnBlur: () => void
+    invalidAppName: boolean
+    invalidAppNameMessage: string
+}
+
+export interface ValueNameInputType {
```

Comment:
Both ValueNameInputType & InputType look the same. Remove either one.

Reply:
Done